server/index.js

```diff
@@ -0,0 +1,77 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const fs = require('fs');
+const db = require('./database.js');
+const path = require('path');
+const port = 3003;
+const app = express();
+
+
+app.use(bodyParser.json());
+app.use(express.static('public'));
+//app.use(express.static(__dirname + '/public'));
+app.use(express.static(__dirname + '/client')); //could be problematic as well
+
+app.get('/getGamesBySeries', function(req, res) {
+  // console.log('data: ');
+  console.log('req params: ' + JSON.stringify(req.params));
+  db.getGamesBySeries(req.params, function(error, result)
+  {
+    if (error)
+    {
+      res.status(500);
+      return;
+    } else {
+      res.status(200).json(result);
+      return;
```

Comment:
I believe you can use res.end() to end the connection rather than return.  Setting the status may also solve the issue but I have typicall used res.end